sdk/python/kfp/compiler/compiler.py

```diff
@@ -57,14 +57,23 @@ def my_pipeline(a: int = 1, b: str = "default value"):
 
   def __init__(
       self,
-      mode: dsl.PipelineExecutionMode = dsl.PipelineExecutionMode.V1_LEGACY):
+      mode: dsl.PipelineExecutionMode = dsl.PipelineExecutionMode.V1_LEGACY,
+      launcher_image: Optional[str] = None):
+    """Creates a KFP compiler for compiling pipeline functions for execution.
+
+    Args:
+      mode: The pipeline execution mode to use.
+      launcher_image: Configurable image for KFP launcher to use. Only applies
+        when `mode == dsl.PipelineExecutionMode.V2_COMPATIBLE`.
```

Comment:
Shall we also mention that this field is only for dev/testing purposes?
(probably for users who want to deploy on-prem, they want to mirror official images, but other than that, users shouldn't set this image)

Reply:
Done.